docs/script-api.md

```diff
@@ -731,6 +731,7 @@ Derived from [`Entity`](#entity) [`Movable`](#movable) [`Monster`](#monster) [`C
 - [`saved_hamsters`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=saved_hamsters) &StateMemory::saved_hamsters
 - [`win_state`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=win_state) &StateMemory::win_state
 - [`illumination`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=illumination) &StateMemory::illumination
+- [`money_total`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=money_total) &StateMemory::money_total
```

Comment:
I believe that this file is generated from generate.py, I think we will want to edit that to update this script-api.

Reply:
This file is indeed auto-generated. Can you clarify what exactly needs to be edited/updated?

Reply:
oh you know what I think this part will get updated, please ignore me :)